runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java

```diff
@@ -282,4 +285,14 @@ private void drawTooltipIfMouseover(Graphics2D graphics, XpGlobe mouseOverSkill,
 		xpTooltip.render(graphics);
 		graphics.translate(-x, -y);
 	}
+
+	private String getActionKillsText(String skillName)
+	{
+		List<Skill> COMBAT = Arrays.asList(Skill.ATTACK, Skill.STRENGTH, Skill.DEFENCE, Skill.RANGED, Skill.HITPOINTS);
+		return COMBAT.stream()
```

Comment:
Although concise, this is overdoing it.. 

You might as well just use an ImmutableSet then do .contains(skill) to check.

On that note, can you not use skillName and actually use the skill enum? It looks like you may have to do some data passing to get it.